### PR TITLE
mintie - set bpipe to v0.9.9.2

### PR DESCRIPTION
**Pull request by mcmero**
MINTIE has issues with newer versions of bpipe. Setting the bpipe version to 0.9.9.2, which is working correctly in tests.

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

File: recipes/mintie/meta.yaml [hunk 1]
```diff
@@ -9,12 +9,12 @@ source:
...
 
 build:
-  number: 0
+  number: 1
   noarch: generic
 
 requirements:
   run:
-    - bpipe
+    - bpipe =0.9.9.2
```
Comment (johanneskoester) [inline recipes/mintie/meta.yaml:17]: Please add a comment that explains the issues or links to them.

**Review by johanneskoester — CHANGES_REQUESTED** [review]
Thanks a lot! One request below.

File: recipes/mintie/meta.yaml (hunk 1 above)
Comment (mcmero) [reply recipes/mintie/meta.yaml:17]: Thanks for reviewing. The original issue is described in https://github.com/Oshlack/MINTIE/issues/26. However, bpipe's author has mentioned incompatibility with JDK17, which is used for the bioconda bpipe builds from 0.9.9.7 onwards. 

I've submitted a pull request to fix this issue for bpipe: https://github.com/bioconda/bioconda-recipes/pull/38762. If this fixes the mintie issues, we can discard this PR.

Comment (mcmero) [reply recipes/mintie/meta.yaml:17]: It looks like my PR for bpipe (https://github.com/bioconda/bioconda-recipes/pull/38762) is stuck. Would it be possible to approve this PR in the meantime to ensure MINTIE installs with a compatible version of bpipe?

**Pull request closed without merging**